otwcode/otwarchive
# AO3-4692 multiple external byline

## Issue

https://otwarchive.atlassian.net/browse/AO3-4692
## Purpose

Fixes a bug which caused only one external author created by an archivist to be displayed on imported works.
## Testing

This also removes duplication in the byline code.
- check that existing stories have the same byline they normally have (including Anonymous)
- check that importing a work as an archivist with an author and co-author who do not have accounts on the Archive displays both authors in the format `name1 [archived by archivist], name2 [archived by archivist]`
